src/transformers/models/opt/modeling_opt.py

```diff
@@ -60,6 +60,15 @@
 _SEQ_CLASS_EXPECTED_OUTPUT = "'LABEL_0'"
 
 
+def _attention_to_position_ids(attention_mask: torch.Tensor) -> torch.Tensor:
+    """
+    Converts an attention mask to position ids.
+    """
+
+    position_ids = torch.cumsum(attention_mask, dim=1)
+    return (position_ids * attention_mask - 1).long()
```

Comment:
Humm let's avoid such one liners

Reply:
OK, now i understand. I will change that! thanks!